### ISSUE TITLE
Deal with failing `system` calls

### ISSUE THREAD

**Issue by pkoch** [tildeclub/welcome#1]
When I was being on-boarded, I noticed that something went wrong with both `passwd` and `chsh`. However, the wizards just charged along, so I kinda just did what it told me (until I hit another brick wall, but I'll consider that out of scope on this ticket).

I think it would make sense to have a local helper that loops while the command doesn't succeed. For the `passwd` case, that should be fine, but I don't know how to deal with the `chsh` one.

Let's maybe just start with a dumb fix for `passwd` and see where that takes us, or do we want to think more high level?

**Comment by premek** [comment]
same here. passwd didn't like my dictionary password and didnt let me re-enter. 

selecting shell didn't work after that:
```
  which shell would you like to use? bash

  great, you've picked /usr/bin/bash!
  in order to change your shell, you'll have to enter your password again
Changing shell for premek.
chsh: Shell not changed.
```
also i think it can be useful to let the user know how to start the firstlogin script again

**Comment by benharri** [comment]
this should be fixed in 83804b2cf85a6fa59069c9b9cf84234f84d5bb61 would anyone like to confirm? i also added a note about how to run the script again
